apps/test/unit/lib/kits/maker/PlaygroundComponentsTest.js

```diff
@@ -36,46 +36,52 @@ describe('Circuit Playground Components', () => {
       // This test is here to warn us if we add a new component but
       // don't cover it with new tests.  If that happens, make sure you
       // add matching tests below!
-      const components = createCircuitPlaygroundComponents(board);
-      expect(Object.keys(components)).to.deep.equal([
-         'colorLeds',
-          'led',
-          'toggleSwitch',
-          'buzzer',
-          'soundSensor',
-          'lightSensor',
-          'tempSensor',
-          'accelerometer',
-          'buttonL',
-          'buttonR',
-          // TODO (captouch): Restore when we re-enable
-          // 'touchPad0',
-          // 'touchPad1',
-          // 'touchPad2',
-          // 'touchPad3',
-          // 'touchPad6',
-          // 'touchPad9',
-          // 'touchPad10',
-          // 'touchPad12',
-      ]);
+      return createCircuitPlaygroundComponents(board).then(components => {
+        expect(Object.keys(components)).to.deep.equal([
+           'colorLeds',
+            'led',
+            'toggleSwitch',
+            'buzzer',
+            'soundSensor',
+            'lightSensor',
+            'tempSensor',
+            'accelerometer',
+            'buttonL',
+            'buttonR',
+            // TODO (captouch): Restore when we re-enable
+            // 'touchPad0',
+            // 'touchPad1',
+            // 'touchPad2',
+            // 'touchPad3',
+            // 'touchPad6',
+            // 'touchPad9',
+            // 'touchPad10',
+            // 'touchPad12',
+        ]);
+      });
     });
```

Comment:
Specifically, here's [the documentation for Mocha promise support](https://mochajs.org/#working-with-promises).  If the promise rejects the test will fail.